### PR TITLE
[CloudKit] Add support for Xcode13 beta1.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/CloudKit/Enums.cs [hunk 1]
```diff
@@ -14,7 +14,9 @@ public enum CKAccountStatus : long {
 		CouldNotDetermine = 0,
 		Available = 1,
 		Restricted = 2,
-		NoAccount = 3
+		NoAccount = 3,
+		[Mac(12,0), iOS(15,0), TV(15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/CloudKit/Enums.cs:18]: minor: space before `(`

but that looks like an error enum... and those, being output only, do not need availability

File: tests/xtro-sharpie/common-CloudKit.ignore [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ## Not intended to be used directly by client code
 !missing-protocol! CKRecordKeyValueSetting not bound
 !missing-protocol-conformance! CKRecord should conform to CKRecordKeyValueSetting (defined in 'CKRecordKeyValueSettingConformance' category)
+!missing-selector! CKRecord::encryptedValues not bound
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/common-CloudKit.ignore:12]: Not to be added due to the same reason, also intro says the type cannot be loaded.

File: src/CloudKit/Enums.cs [hunk 3]
```diff
@@ -82,6 +84,8 @@ public enum CKErrorCode : long {
 		ParticipantMayNeedVerification = 33,
 		ResponseLost = 34,
 		AssetNotAvailable = 35,
+		[Mac(12,0), iOS(15,0), TV(15,0), MacCatalyst (15,0)]
```
Comment (spouliot) [inline src/CloudKit/Enums.cs:87]: not needed

File: src/CloudKit/Enums.cs [hunk 4]
```diff
@@ -127,7 +131,11 @@ public enum CKQueryNotificationReason : long {
 	public enum CKRecordZoneCapabilities : ulong {
 		FetchChanges = 1 << 0,
 		Atomic = 1 << 1,
-		[iOS (10,0), TV (10,0), Mac (10,12)] Sharing = 1 << 2,
+		[iOS (10,0), TV (10,0), Mac (10,12)]
+		Sharing = 1 << 2,
+		[Mac(12,0), iOS(15,0), TV(15,0)]
```
Comment (spouliot) [inline src/CloudKit/Enums.cs:136]: minor: space before `(`

File: src/cloudkit.cs [hunk 5]
```diff
@@ -115,6 +119,10 @@ interface CKShareMetadata : NSCopying, NSSecureCoding
 
 		[NullAllowed, Export ("rootRecord", ArgumentSemantic.Strong)]
 		CKRecord RootRecord { get; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:123]: minor: space before `(`

File: src/cloudkit.cs [hunk 6]
```diff
@@ -129,6 +137,11 @@ interface CKShare
 		[DesignatedInitializer]
 		IntPtr Constructor (CKRecord rootRecord, CKRecordID shareID);
 
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:140]: minor: space before `(`

File: src/cloudkit.cs [hunk 7]
```diff
@@ -457,6 +470,10 @@ interface CKErrorFields {
 
 		[Field ("CKErrorRetryAfterKey")]
 		NSString ErrorRetryAfterKey { get; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:474]: catalyst ?

File: src/cloudkit.cs [hunk 8]
```diff
@@ -575,6 +592,9 @@ interface CKFetchRecordChangesOperation {
 	[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12)]
 	delegate void CKFetchRecordZoneChangesFetchCompletedHandler (CKRecordZoneID recordZoneID, CKServerChangeToken serverChangeToken, NSData clientChangeTokenData, bool moreComing, NSError recordZoneError);
 
+	[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:595]: catalyst ?

File: src/CloudKit/Enums.cs (hunk 1 above)
Comment (mandel-macaque) [inline src/CloudKit/Enums.cs:18]:
```suggestion
		[Mac (12,0), iOS (15,0), TV (15,0), MacCatalyst (15,0)]
```

File: src/CloudKit/Enums.cs (hunk 3 above)
Comment (mandel-macaque) [inline src/CloudKit/Enums.cs:87]:
```suggestion
		[Mac (12,0), iOS (15,0), TV (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 9]
```diff
@@ -897,12 +952,28 @@ interface CKModifyRecordsOperation {
...
 		[Export ("atomic")]
 		bool Atomic { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
+		[Export ("perRecordSaveBlock", ArgumentSemantic.Copy)]
+		CKModifyRecordsOperationPerRecordSaveHandler PerRecordSaveHandler { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:962]: missing space before `(`

File: src/CloudKit/Enums.cs (hunk 4 above)
Comment (mandel-macaque) [inline src/CloudKit/Enums.cs:136]:
```suggestion
		[Mac (12,0), iOS (15,0), TV (15,0)]
```

File: src/cloudkit.cs [hunk 10]
```diff
@@ -897,12 +952,28 @@ interface CKModifyRecordsOperation {
...
+		[Export ("perRecordDeleteBlock", ArgumentSemantic.Copy)]
+		CKModifyRecordsOperationPerRecordDeleteHandler PerRecordDeleteHandler { get; set; }
 		
 	}
 
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKModifyRecordZonesHandler (CKRecordZone [] savedRecordZones, CKRecordZoneID [] deletedRecordZoneIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:971]: catalyst ?

File: src/cloudkit.cs [hunk 11]
```diff
@@ -929,11 +1000,27 @@ interface CKModifyRecordZonesOperation {
...
 			set; 
 		}
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
+		[Export ("perRecordZoneSaveBlock", ArgumentSemantic.Copy)]
+		CKModifyRecordZonesPerRecordZoneSaveHandler PerRecordZoneSaveHandler { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1010]: missing space before `(1

File: src/cloudkit.cs [hunk 12]
```diff
@@ -739,12 +771,20 @@ interface CKFetchRecordZonesOperation {
 		[Static]
 		[Export ("fetchAllRecordZonesOperation")]
 		CKFetchRecordZonesOperation FetchAllRecordZonesOperation ();
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:776]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 13]
```diff
@@ -929,11 +1000,27 @@ interface CKModifyRecordZonesOperation {
...
+		[Export ("perRecordZoneDeleteBlock", ArgumentSemantic.Copy)]
+		CKModifyRecordZonesPerRecordZoneDeleteHandler PerRecordZoneDeleteHandler { get; set; }
 	}
 
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKModifySubscriptionsHandler (CKSubscription [] savedSubscriptions, string [] deletedSubscriptionIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1018]: catalyst ?

File: src/cloudkit.cs [hunk 14]
```diff
@@ -961,6 +1048,16 @@ interface CKModifySubscriptionsOperation {
 			get;
 			set; 
 		}
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1053]: missing space before `(`

File: src/cloudkit.cs [hunk 15]
```diff
@@ -961,6 +1048,16 @@ interface CKModifySubscriptionsOperation {
...
+		[Export ("perSubscriptionSaveBlock", ArgumentSemantic.Copy)]
+		CKModifySubscriptionsPerSubscriptionSaveHandler PerSubscriptionSaveHandler { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1058]: missing space before `(`

File: src/cloudkit.cs (hunk 5 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:123]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 16]
```diff
@@ -1257,6 +1354,9 @@ interface CKQuery : NSSecureCoding, NSCopying {
 		NSSortDescriptor [] SortDescriptors { get; set; }
 	}
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1357]: catalyst ?

File: src/cloudkit.cs [hunk 17]
```diff
@@ -1293,6 +1393,10 @@ interface CKQueryOperation {
 		[Export ("desiredKeys", ArgumentSemantic.Copy)][NullAllowed]
 		string [] DesiredKeys { get; set; }
 
+		[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use 'RecordMatchedHandler' instead.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Use 'RecordMatchedHandler' instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use 'RecordMatchedHandler' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'RecordMatchedHandler' instead.")]
```
Comment (spouliot) [inline src/cloudkit.cs:1399]: catalyst ?

File: src/cloudkit.cs (hunk 7 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:474]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 18]
```diff
@@ -1306,6 +1410,11 @@ interface CKQueryOperation {
 			get;
 			set; 
 		}
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1415]: missing space before `(`

File: src/cloudkit.cs (hunk 8 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:595]:
```suggestion
	[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 19]
```diff
@@ -627,6 +651,11 @@ interface CKFetchRecordZoneChangesOperation
 
 		[NullAllowed, Export ("fetchRecordZoneChangesCompletionBlock", ArgumentSemantic.Copy)]
 		Action<NSError> ChangesCompleted { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:656]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 20]
```diff
@@ -1338,6 +1449,10 @@ interface CKRecord : NSSecureCoding, NSCopying {
 		[Field ("CKRecordTypeShare")]
 		NSString TypeShare { get; }
 
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1452]: catalyst ?

File: src/cloudkit.cs [hunk 21]
```diff
@@ -713,6 +742,9 @@ interface CKFetchRecordsOperation {
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKRecordZoneCompleteHandler (NSDictionary recordZonesByZoneId, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:745]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 22]
```diff
@@ -739,12 +771,20 @@ interface CKFetchRecordZonesOperation {
...
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
+		[Export ("perRecordZoneCompletionBlock", ArgumentSemantic.Copy)]
+		CKRecordZonePerRecordZoneCompletionHandler PerRecordZoneCompletionHandler { get; set; }
 	}
 
 	[NoWatch]
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKFetchSubscriptionsCompleteHandler (NSDictionary subscriptionsBySubscriptionId, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:785]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 23]
```diff
@@ -1457,6 +1572,10 @@ interface CKRecordZone : NSSecureCoding, NSCopying {
 		[Static]
 		[Export ("defaultRecordZone")]
 		CKRecordZone DefaultRecordZone ();
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1576]: do a find and replace :)

File: src/cloudkit.cs [hunk 24]
```diff
@@ -1764,6 +1883,9 @@ interface CKDiscoverAllUserIdentitiesOperation
 		Action<NSError> Completed { get; set; }
 	}
 
+	[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1886]: catalyst ?

File: src/cloudkit.cs [hunk 25]
```diff
@@ -772,6 +812,11 @@ interface CKFetchSubscriptionsOperation {
 		[Static]
 		[Export ("fetchAllSubscriptionsOperation")]
 		CKFetchSubscriptionsOperation FetchAllSubscriptionsOperation ();
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:817]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 26]
```diff
@@ -847,6 +892,12 @@ interface CKModifyBadgeOperation {
 	[iOS (8,0), Mac (10,10), Watch (3,0)]
 	delegate void CKModifyRecordsOperationHandler (CKRecord [] savedRecords, CKRecordID [] deletedRecordIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:895]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 27]
```diff
@@ -1780,11 +1902,20 @@ interface CKFetchShareParticipantsOperation
 		[Export ("userIdentityLookupInfos", ArgumentSemantic.Copy)]
 		CKUserIdentityLookupInfo[] UserIdentityLookupInfos { get; set; }
 
+		[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use 'PerShareParticipantCompletionHandler' instead.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Use 'PerShareParticipantCompletionHandler' instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use 'PerShareParticipantCompletionHandler' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'PerShareParticipantCompletionHandler' instead.")]
```
Comment (spouliot) [inline src/cloudkit.cs:1908]: catalyst ?

File: src/cloudkit.cs [hunk 28]
```diff
@@ -1780,11 +1902,20 @@ interface CKFetchShareParticipantsOperation
...
 		[NullAllowed, Export ("shareParticipantFetchedBlock", ArgumentSemantic.Copy)]
 		Action<CKShareParticipant> Fetched { get; set; }
 
 		[NullAllowed, Export ("fetchShareParticipantsCompletionBlock", ArgumentSemantic.Copy)]
 		Action<NSError> Completed { get; set; }
+
+		[NullAllowed]
+		[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:1916]: f&r

File: src/cloudkit.cs [hunk 29]
```diff
@@ -847,6 +892,12 @@ interface CKModifyBadgeOperation {
 	[iOS (8,0), Mac (10,10), Watch (3,0)]
 	delegate void CKModifyRecordsOperationHandler (CKRecord [] savedRecords, CKRecordID [] deletedRecordIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	delegate void CKModifyRecordsOperationPerRecordSaveHandler (CKRecordID recordId, CKRecord record, NSError error);
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:898]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 30]
```diff
@@ -897,12 +952,28 @@ interface CKModifyRecordsOperation {
 
 		[Export ("atomic")]
 		bool Atomic { get; set; }
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:957]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 31]
```diff
@@ -1885,5 +2016,9 @@ interface CKFetchDatabaseChangesOperation
 
 		[NullAllowed, Export ("fetchDatabaseChangesCompletionBlock", ArgumentSemantic.Copy)]
 		CKFetchDatabaseChangesCompletionHandler ChangesCompleted { get; set; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (spouliot) [inline src/cloudkit.cs:2020]: f&r

File: src/cloudkit.cs (hunk 9 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:962]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 10 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:971]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 32]
```diff
@@ -897,12 +952,28 @@ interface CKModifyRecordsOperation {
...
 		
 	}
 
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKModifyRecordZonesHandler (CKRecordZone [] savedRecordZones, CKRecordZoneID [] deletedRecordZoneIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	delegate void CKModifyRecordZonesPerRecordZoneSaveHandler (CKRecordZoneID zoneId, CKRecordZone zone, NSError error);
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:974]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: tests/xtro-sharpie/common-CloudKit.ignore (hunk 2 above)
Comment (spouliot) [inline tests/xtro-sharpie/common-CloudKit.ignore:12]: is the comment above applicable here ?

File: src/cloudkit.cs [hunk 33]
```diff
@@ -929,11 +1000,27 @@ interface CKModifyRecordZonesOperation {
 			get;
 			set; 
 		}
+
+		[NullAllowed]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst(15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:1005]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

**Review by spouliot — CHANGES_REQUESTED** [review]

File: src/cloudkit.cs (hunk 11 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1010]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 13 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1018]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs [hunk 34]
```diff
@@ -929,11 +1000,27 @@ interface CKModifyRecordZonesOperation {
...
+		CKModifyRecordZonesPerRecordZoneDeleteHandler PerRecordZoneDeleteHandler { get; set; }
 	}
 
 	[iOS (8,0), Mac (10,10)]
 	delegate void CKModifySubscriptionsHandler (CKSubscription [] savedSubscriptions, string [] deletedSubscriptionIds, NSError operationError);
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	delegate void CKModifySubscriptionsPerSubscriptionSaveHandler (NSString subscriptionId, CKSubscription subscription, NSError error);
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```
Comment (mandel-macaque) [inline src/cloudkit.cs:1021]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 14 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1053]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 15 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1058]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 16 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1357]:
```suggestion
	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 18 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1415]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 20 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1452]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 23 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1576]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 24 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1886]:
```suggestion
	[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 28 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:1916]:
```suggestion
		[iOS (15,0), Watch (8,0), TV (15,0), Mac (12,0), MacCatalyst (15,0)]
```

File: src/cloudkit.cs (hunk 31 above)
Comment (mandel-macaque) [inline src/cloudkit.cs:2020]:
```suggestion
		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

File: tests/xtro-sharpie/common-CloudKit.ignore (hunk 2 above)
Comment (mandel-macaque) [reply tests/xtro-sharpie/common-CloudKit.ignore:12]: yes, same issue.

**Review by spouliot — APPROVED** [review]

**Pull request merged**